.circleci/config.yml

```diff
@@ -20,6 +21,7 @@ commands:
           # xz: circleci checkout steps
           command: |
             nix-env -iA \
+              nixpkgs.awscli \
```

Comment:
This shouldn't be necessary, there's already a binstub for the aws cli.
```suggestion
```

Reply:
There is in the `expo/universe` repo but not here, unless I missed sth.

Reply:
https://github.com/expo/expo/blob/master/bin/aws

Reply:
This is PR for sdk-32 branch (I'm not sure why) and aws cli is not available there. Dominik will be back on Monday, so I'll probably leave this to him.